Sources/DataPipeline/Resources/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<!--
+	Docs for this section: https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_data_use_in_privacy_manifests
+	This section is describing what information our SDK collects about the app user. 
+	
+	It's my understanding that this section is for what our SDK requires, not what a customer can do with our SDK. Therefore, if something is optional, use `false`.
+	Customers must create their own privacy file for what their app does and provide that to Apple. That's where customers report what they might be using our SDK for. 
```

Comment:
Customers do not need to declare their dependancies privacy requirements based on the second line in this call out at the top of [this page](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_data_use_in_privacy_manifests).
![image](https://github.com/customerio/customerio-ios/assets/153670820/d1a953fe-682f-4dac-9fbf-1c98fa77f547)

Reply:
Re-reviewing the comments in this file and the documentation you linked to, I think the comments in this file are up-to-date and accurate. 

Do you have a suggestion of a change to make to these comments?

Reply:
I think the issue is around the interpretation of "required". Since their app is collecting an email address and passing it to us, they would have to declare that. If we ever created a UI to collect the email and link to a user then we'd have to declare the email address. Therefore in the first case email isn't a "required" part of our SDK. 

So this does work, but the wording made that tough to interpret. I provided a suggestion to the line that might make that more clear.